man/leapp.1

```diff
@@ -1,59 +1,165 @@
+
+\fB--whitelist-experimental\fR <\fIActorName\fR>
+    Enable an experimental actor. Can be used multiple times.
+    To use this variable, \fBLEAPP_UNSUPPORTED\fR has to be set. See \fBDeveloper variables\fR for more information.
+
+\fB--reboot\fR
+    (upgrade only) Automatically perform reboot when requested.
+
+\fB--resume\fR
+    (upgrade only) Continue the last execution after it was stopped (e.g. after reboot).
+
+.SS answer
+\fB--add\fR
+    If set, sections will be created even if missing in original answerfile.
+
+\fB--section\fR <\fIdialog_sections\fR>
+    Register answer for a specific section in the answerfile. Can be used multiple times.
+
+
+.SH ENVIRONMENT
+If the argument for the environment variables below is not specified, it is possible to set them to either 1 (true) or 0 (false). They default to 0.
+
+.B LEAPP_CONFIG
+.RS 4
+Overrides the default location of \fIleapp.conf\fR. If not specified, \fI.leapp/leapp.conf\fR is used when the command is executed inside a leapp repository, otherwise the default \fI/etc/leapp/leapp.conf\fR is used.
+.RE
+
+.B LEAPP_LOGGER_CONFIG
+.RS 4
+Overrides the default location of \fIlogger.conf\fR. If not specified, the default \fI/etc/leapp/logger.conf\fR is used.
+.RE
+
+.B LEAPP_DEBUG
+.RS 4
+Enables debug logging. Equivalent to \fB--debug\fR, which takes precedence.
+.RE
+
+.B LEAPP_VERBOSE
 .RS 4
-This environment variable allows to override the default location of leapp.conf. If not specified, .leapp/leapp.conf is used when the command is executed inside a leapp repository, otherwise the default /etc/leapp/leapp.conf is used.
+Enables debug logging. Equivalent to \fB--verbose\fR, which takes precedence.
 .RE
 
-LEAPP_LOGGER_CONFIG
+.B LEAPP_GRUB_DEVICE
 .RS 4
-This environment variable allows to override the default location of logger.conf. If not specified, the default /etc/leapp/logger.conf is used.
+Overrides the automatically detected GRUB core location.
 .RE
 
-LEAPP_DEBUG
+.B LEAPP_NO_RHSM
 .RS 4
-This environment variable enables debug logging, see --debug for more information
+Skip actions that use Red Hat Subscription Manager. Equivalent to \fB--no-rhsm\fR, which takes precedence.
 .RE
 
-LEAPP_VERBOSE
+.B LEAPP_OVL_SIZE
 .RS 4
-This environment variable enables verbose logging, see --verbose for more information
+For each XFS partition created with \fIftype=0\fR, leapp creates an overlay file in order to proceed. This option sets the size (in MB) of every such file. Defaults to \fB2048\fR.
 .RE
 
-.SH "RETURN CODES"
 
-0 - No error occurred.
+.SS Developer variables
+These variables shouldn't be needed under normal circumstances and aren't meant for normal use. 
+
+.B LEAPP_UNSUPPORTED
+.RS 4
+Necessary in case you use any variable starting with \fBLEAPP_DEVEL\fR (see below) or use \fB--whitelist-experimental\fR.
+
+By setting this variable to 1, you essentially acknowledge that some options you might use are not fully supported and not meant for ordinary users.
```

Comment:
```suggestion
By setting this variable to 1, you acknowledge that the upgrade is not going to be supported by Red Hat.
```
You mind offend somebody when calling them ordinary :)

Reply:
@shaded-enmity, the whole upgrade becomes unsupported if LEAPP_UNSUPPORTED is used, right?

Reply:
@bocekm yes